tests/test_kms/test_model.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+
+from moto.kms.models import KmsBackend
+
+PLAINTEXT = b"text"
+REGION = "us-east-1"
+
+
+@pytest.fixture
+def backend():
+    return KmsBackend(REGION)
+
+
+@pytest.fixture
+def key(backend):
+    return backend.create_key(
+        None, "ENCRYPT_DECRYPT", "SYMMETRIC_DEFAULT", "Test key", None, REGION
+    )
+
+
+def test_encrypt_key_id(backend, key):
+    ciphertext, arn = backend.encrypt(key.id, PLAINTEXT, {})
+
+    assert ciphertext is not None
+    assert arn is not None
```

Comment:
These assertions should be more specific. What is the format of the ARN that we're expecting?

(Same goes for all other tests)